server/controllers/finance/exchange.js

```diff
@@ -16,13 +16,14 @@ exports.list = function list(req, res, next) {
       enterprise = req.session.enterprise;
   
   sql =
-    'SELECT currency_id, rate, date ' +
+    'SELECT id, currency_id, rate, date ' +
     'FROM exchange_rate ' + 
     'WHERE enterprise_id = ? ' +
     'ORDER BY date;';
 
   db.exec(sql, [ enterprise.currency_id ])
```

Comment:
The column must contain the enterprise_id.  If it doesn't, you should change it to contain the enterprise id.